marbles/core/marbles/core/marbles.py

```diff
@@ -330,13 +330,18 @@ def formattedMsg(self):  # mimic unittest's name for standardMsg
 
     @classmethod
     def _format_local(cls, name, value):
+        def truncate(s):
+            return unittest.util.safe_repr(s, short=True)
```

Comment:
I ended up doing this to make the lines shorter. Not sure if this is good or not.

Reply:
I prefer not defining nested functions like this. There are a couple of other ways to shorten the lines here:

1. Import `unittest.util` directly so you can call `util.safe_repr` (which is a good deal shorter than `unittest.util.safe_repr`)
```python
from unittest import util
```
2. Use line breaking and even more concise variable names!
```python
short_lines = [unittest.util.safe_repr(line, short=True)
               for line in value_str.split('\n')]
short_value_str = '\n'.join(short_lines)
value_str = textwrap.indent(short_value_str, '\t\t')
``` 

I want to think more about what exactly we want to do with multiline locals, but since you asked about this piece of code specifically I figured I'd show you these options.

Reply:
Great, that makes sense. Thanks!
Are there conventions on when/how to line break? I know many use black to auto format the code.

Reply:
[PEP8](https://www.python.org/dev/peps/pep-0008/) has a section about breaking long lines, but it doesn't have a ton of examples. The Hitchiker's Guide to Python has a short section on [Line Continuations](https://docs.python-guide.org/writing/style/#line-continuations), too, but again, not a ton of examples.

One of the best things you can do is simply to read Python code and see how long lines are broken. You're more than welcome to read more of marbles the source code to get a sense of our line-breaking style.

You can also use black, and see how it breaks up long lines in different scenarios. We don't use black to format marbles, so running black on the whole codebase may change a lot of things outside of the specific code you're changing here.

Reply:
Got it!